### ISSUE TITLE
crop_multichannel_data fails when crop_size is None

### ISSUE THREAD

**Issue by geneva-miller** [vanvalenlab/caliban-toolbox#122]
In reshape_data.crop_multichannel_data, `crop_size = None` causes `log_data['row_crop_size'] = crop_size[0]` to raise an error. This makes it impossible to actually use `crop_nums` as the cropping parameter instead of `crop_size`. `log_data['row_crop_size']` and `log_data['col_crop_size']` don't appear to be used in crop_utils.stitch_crops, are these values used anywhere else? Perhaps we could just remove those entries from `log_data`.

**Comment by ngreenwald** [comment]
Good point, looks like we're not testing the high-level function with crop_size=None, which is how the bug slipped in. `col_crop_size` is used in a couple places throughout the repo to reconstruct the original image, so I don't think we can remove it, we'll have to add it in.
